tests/test_expectation.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+# $VENV/bin/pytest tests/test_expectation.py -q
+
+
+@pytest.mark.parametrize("test_input,expected", [(1, "PAVICS"), (2, "UofT")])
+class NodeFunctionalTestsPyTest:
+
+    # If the table is created, test if the information for a specific node is as expected
+    # Uses sample information from the node registry
+    def test_node_info_view(self, test_input, expected):
+        import sys
+        from sqlalchemy import engine_from_config
+        from pyramid.paster import (
+            get_appsettings,
+            setup_logging,
+        )
+        from network_manager.initialize_db import table_exists
+
+        argv = sys.argv
```

Comment:
This was when sqlite was used as the database and I think it could not create the engine without the development.ini filename being passed to it.

Reply:
No, I think when I ran this test I got an error saying the engine could not be created and it was expecting an argument (I think).  Since in earlier code the engine was created through initializing by commandline I thought  the commandline was what was needed.

Reply:
Let's discuss this in person/zoom please